src/components/profile/components/Profile.jsx

```diff
@@ -4,6 +4,7 @@ import Grid from '@material-ui/core/Grid';
 import { Navigation } from '../../Navigation';
 import { Experts } from '../../feed/components';
 import Overview from './Overview';
+import About from './About';
```

Comment:
Something is off with the importing order import About should be before overview